packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTag.component.js

```diff
@@ -51,12 +51,10 @@ export default class MultiSelectTag extends React.Component {
 	 * On Tags value change, we update suggestions if they are displayed
 	 * @param { Object } nextProps The new props
 	 */
-	UNSAFE_componentWillReceiveProps(nextProps) {
-		if (nextProps.value === this.props.value) {
-			return;
-		}
-		if (this.state.suggestions) {
-			this.updateSuggestions(undefined, nextProps);
+	componentDidUpdate(prevProps) {
+		const hasSuggestions = this.state.suggestions;
+		if (hasSuggestions && prevProps.value !== this.props.value) {
```

Comment:
The variable is not reused
```suggestion
		if (this.state.suggestions && prevProps.value !== this.props.value) {
```